Throw error if scale file is missing for GemNet-OC

abhshkdz opened the pull request:
The scale file is quite critical for GemNet-OC, especially when running inference from a pretrained checkpoint. This change helps avoid inadvertent mispredictions.

I've also updated all released GemNet-OC checkpoints to have the correct model name and scale file path (#379).

AdeeshKolluru commented:
LGTM, thanks! We might also want to add the configs link and updated checkpoints [here](https://github.com/Open-Catalyst-Project/ocp/blob/main/MODELS.md).

abhshkdz commented:
> .. and updated checkpoints [here](https://github.com/Open-Catalyst-Project/ocp/blob/main/MODELS.md).

The checkpoint URLs are the same as before. I added URLs to the configs and scale files.

zulissi reviewed (approved):
This is great, thanks!
